Makefile

```diff
@@ -368,6 +379,7 @@ clear-stalled:
 		echo $${PS_OUT} | xargs kill; \
 	fi
 
+# GC test
```

Comment:
This comment doesn't really convey any additional information than can be inferred from the target name.